src/ResourceManager/Network/Commands.Network/VirtualNetwork/Subnet/AzureVirtualNetworkSubnetConfigBase.cs

```diff
@@ -58,5 +59,11 @@ public class AzureVirtualNetworkSubnetConfigBase : NetworkBaseCmdlet
             ParameterSetName = "SetByResource",
             HelpMessage = "RouteTable")]
         public PSRouteTable RouteTable { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Private Access Service")]
+        public List<string> PrivateAccessService { get; set; }
```

Comment:
@EvgenyAgafonchikov is there a set of predetermined strings that can be provided to this parameter, or is there no bound?